.github/workflows/autobuild.yml

```diff
@@ -43,6 +44,7 @@ on:
       - '**README.md'
       - 'docs/**'
       - 'SECURITY.md'
+      - 'CONTRIBUTING.md'
```

Comment:
Different list order than the list above now... :)

Reply:
Thanks. Fixed